_learning_hours/legacy/domain_observability.md

```diff
@@ -0,0 +1,108 @@
+---
+theme: refactoring
+title: Domain Oriented Obervability on Reservation Kata
+name: domain_oriented_observability
+kata: reservation
+difficulty: 2
+author: pierrickblons
+---
+
+# Domain Oriented Obervability
+
+It's useful to have logging and metrics in our code to understand how it's behaving in production. But sometimes those little calls to loggers or metrics class start to spread all across the code base. It's quite common to see methods with more code dedicated to observability than code actually solving the problem.
+
+In this learning hour we will see how to improve the understandability of code that requires observability.
+
+### Learning Goals
+
+- Improving the readability of the code
```

Comment:
Learning goals should begin with a verb from the "Bloom taxonomy" - I would suggest  "recognize when a domain probe would increase code readability" and "use the extract method refactoring"